tests/CliTest.php

```
@@ -38,6 +38,40 @@ public function testWorkingMemoryLifecycleStillWorks(): void
+        self::assertSame(0, $this->invoke([
+            'checkpoint', $id,
+            '--title=review blindspots',
+            '--body=Equals-form values are parsed instead of disappearing.',
+            '--root=' . $this->root,
+        ]));
+
+        $session = $store->load($this->root, $id);
+        $checkpoint = $session->checkpoints[count($session->checkpoints) - 1] ?? null;
+        self::assertIsArray($checkpoint);
+        self::assertSame('review blindspots', $checkpoint['title']);
+        self::assertSame('Equals-form values are parsed instead of disappearing.', $checkpoint['body']);
```

PHPStan (2.2.8)

[error] Offset 'body' does not exist on array{id: string, title: 'review blindspots', created_at: string}.


(offsetAccess.notFound)